specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/azureFirewall.json

```diff
@@ -274,7 +274,7 @@
             },
             "provisioningState":{
                "description":"The provisioning state of the resource.",
-               "$ref":"#/definitions/ProvisioningState"
+               "$ref":"./network.json#/definitions/ProvisioningState"
```

Comment:
Isn't it a breaking change on SDK level? we should not accept any breaking changes on already published API version

Reply:
Removed duplicate definition and linked original. No change in SDK code